integration/spark-common/src/main/scala/org/apache/carbondata/events/AlterTableEvents.scala

```diff
@@ -133,25 +133,25 @@ case class AlterTableRenameAbortEvent(carbonTable: CarbonTable,
 /**
  *
  * @param carbonTable
- * @param carbonLoadModel
+ * @param carbonMergerMapping
```

Comment:
remove all these parameter comment if there is no description

Reply:
ok